src/content/docs/en/vs-workspace-setup.md

```diff
@@ -0,0 +1,133 @@
++ Go to File > Preferences > Settings
++ Choose Workspace
++ Search for the setting you want to modify and adjust its value
+
+### Open workspace settings (JSON)
++ Press Ctrl + Shift + P on your keyboard
++ Search 'Open Workspace Settings (JSON)' option
++ Edit the file that was opened
+
+
+![Open workspace settings](/gifs/open-workspace-settings.gif)
+
+## Setting up your workspace
+
+### Clone the boilerplate
+Please clone the boilerplate by clicking on the following [link](https://github.com/HackMort/jdk-boilerplate).
+
+![Git clone repository](/gifs/git-clone-repo.gif)
+
+### Install boilerplate dependencies
+After cloning the boilerplate install its dependencies by running the following command in your terminal
+```
+npm install
+```
+
+### Install VS Code extensions
+To ensure that your environment can be properly configured to work with the boilerplate, please install the following extensions.
++ [Astro](https://marketplace.visualstudio.com/items?itemName=astro-build.astro-vscode)
++ [ESLint](https://marketplace.visualstudio.com/items?itemName=dbaeumer.vscode-eslint)
++ [Prettier](https://marketplace.visualstudio.com/items?itemName=esbenp.prettier-vscode)
++ [Prettier-Standard](https://marketplace.visualstudio.com/items?itemName=numso.prettier-standard-vscode)
++ [Stylelint](https://marketplace.visualstudio.com/items?itemName=stylelint.vscode-stylelint)
+
+### Add custom settings to your workspace
+Now it's time to configure your workspace so that VS Code can work in conjunction with the installed extensions and dependencies.  
+
+![Edit workspace settings](/gifs/configuring-workspace.gif)
+
+Add following settings in your JSON:
++ Configuration settings to define which formatter handles each type of file. In general, Prettier will be used, Prettier-Standard will handle JavaScript files.
+```json
+  "editor.defaultFormatter": "esbenp.prettier-vscode",
+  "[javascript]": {
+    "editor.defaultFormatter": "numso.prettier-standard-vscode"
+  },
+  "prettier.documentSelectors": ["**/*.astro"],
+  "[astro]": {
+    "editor.defaultFormatter": "esbenp.prettier-vscode"
+  },
+  "prettier.configPath": ""
+```
+
++ To ensure that ESLint is validating our JavaScript files, we add the following to our configuration.
+
+```json
+  "eslint.validate": [
+    "javascript",
+    "javascriptreact",
+    "typescript",
+    "typescriptreact"
+  ]
+```
++ The configuration for enabling Stylelint to handle the validation and formatting of files upon saving in VS Code would be as follows.
+
+```json
+  "css.validate": false,
+  "less.validate": false,
+  "scss.validate": false,
+  "stylelint.enable": true,
+  "stylelint.validate": ["css", "less", "postcss", "scss"],
+  "editor.codeActionsOnSave": {
+    "source.fixAll.stylelint": true
+  }
+```
+
+## Do you want to learn more about related topics?
+You may consider reading the page on this website about the [Boilerplate](/en/boilerplate) or the page specifically dedicated to [Stylelint](/en/stylelint).
```

Comment:
Change to:

We recommend you to take a look at these 2 pages where you can learn more about our Boilerplate and Code Styles we follow:

List:
boilerplate
eslint
stylint